atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -193,13 +193,24 @@ def run(self):
         except AttributeError:
             commit_id = ""
 
-        try:
-            base_image_id = self.workflow.base_image_inspect['Id']
-        except KeyError:
+        base_image = self.workflow.builder.base_image
+        if base_image is not None:
+            base_image_name = base_image.to_str()
+            try:
+                base_image_id = self.workflow.base_image_inspect['Id']
+            except KeyError:
+                base_image_id = ""
+        else:
+            base_image_name = ""
             base_image_id = ""
 
+        try:
+            dockerfile_contents = open(self.workflow.builder.df_path).read()
```

Comment:
Should we use `with open(...) as ...` here?

Reply:
Existing code, just moved around, but sure, I'll squeeze a fix for that in. (Will just include the fix in the patch, rather splitting it out, I think.)